### PR TITLE
Grammatical review of the README

### PR DESCRIPTION
**Pull request by skyzyx**
* Replace LICENSE.txt with LICENSE.md.
* Performed a grammatical review of the README.
    * Cleaned-up use of linking back to source projects.
    * Synchronized how we refer to projects by name.
    * Reformatted some text to improve readability/scanability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,26 @@
-# kafka-connect-bigquery
+# Kafka-BigQuery Connector
```
Comment (C0urante) [inline README.md]: We'd like to stick with the Confluent idiom of "kafka-connect-foo" for directory/project names and "Kafka Connect <Foo> Connector" for referring to the connector itself (see [kafka-connect-hdfs](https://github.com/confluentinc/kafka-connect-hdfs) and [kafka-connect-jdbc](https://github.com/confluentinc/kafka-connect-jdbc)). Please change to "Kafka Connect BigQuery Connector".

File: README.md [hunk 2]
```diff
@@ -25,160 +31,214 @@ datasets=.*=
...
-**NOTE**: You must have Docker installed and running on your machine in order to run integration
+> **NOTE**: You must have [Docker] installed and running on your machine in order to run integration
 tests for the connector.
 
 ### How Integration Testing Works
 
-Integration tests run by creating Docker instances for Zookeeper, Kafka, Schema Registry, and the
-Kafka-BigQuery connector itself, then verifying the results using a Junit test.
+Integration tests run by creating [Docker] instances for [Zookeeper], [Kafka], [Schema Registry], 
+and the Kafka-BigQuery connector itself, then verifying the results using a [JUnit] test.
```
Comment (C0urante) [inline README.md]: Recommend changing "Kafka-BigQuery connector" to "BigQuery Connector".

**Comment by C0urante** [comment]
The attention to detail here is really impressive. Got one nit about how the project/connector is referred to, but apart from that everything looks great.

**Comment by skyzyx** [comment]
Once the checks clear, please re-review.

Thanks!

**Comment by C0urante** [comment]
Looks good to me. If it's a small change, we could wait for the Maven Central badge requested in https://github.com/wepay/kafka-connect-bigquery/issues/17 to be added to this PR, or if you'd rather merge this now and start a new one for the Maven badge, we could do that too. Let me know what you'd like to do!

**Comment by skyzyx** [comment]
I'll add that in a separate PR.

**Pull request merged**